### PR TITLE
Final verification of pollinisation branch : Add title of parties,  remove the part that we don't need anymore

### PR DESCRIPTION
**Pull request by julielgdr**

### CODE REVIEW

File: OCR_pollen.qmd [hunk 1]
```diff
@@ -214,7 +275,7 @@ anova(poll1,poll2)
 
 The P-value is not significant at the 5% threshold, so we reject the hypothesis that grazing has an effect on the RatioPoll. The Anova test confirms our graphical observation: there is no difference between the grazed and natural plots.
 
-##TEST MODEL
```
Comment (Alexia-Gueguen) [inline OCR_pollen.qmd]: 2 titles : remove 1

File: OCR_pollen.qmd [hunk 2]
```diff
@@ -194,11 +255,11 @@ df <- read_delim("pollen.csv", delim = ";",
 ```
 
 ## VISUALISATION OF DATA AND ONE-FACTOR ANALYSIS OF VARIANCE
-
```
Comment (Alexia-Gueguen) [inline OCR_pollen.qmd:250]: remove the "-"

File: OCR_pollen.qmd [hunk 3]
```diff
@@ -7,17 +7,78 @@ output:
...
 ```{=html}
 <style>
 body {
 text-align: justify}
 </style>
 ```
 
-# PARTIE 3: GRAZING EFFECT ON POLLINISATION STRATEGIES 
-packages
-```{r pollen_read_data}
```
Comment (Alexia-Gueguen) [inline OCR_pollen.qmd]: this part is not at the good place

File: OCR_pollen.qmd [hunk 4]
```diff
@@ -7,17 +7,78 @@ output:
   html_document:
    toc: true
    toc_float: true
----
```
[the remark this answers is not in the document]
Comment (MarieEtienne) [reply OCR_pollen.qmd]: No, please do not do that you will break the whole workflow. the yml header (the part starting with --- and finishing with the --- should not be touched

File: OCR_pollen.qmd [hunk 5]
```diff
@@ -7,17 +7,78 @@ output:
...
 body {
 text-align: justify}
 </style>
 ```
 
-# PARTIE 3: GRAZING EFFECT ON POLLINISATION STRATEGIES 
-packages
-```{r pollen_read_data}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+                      INTRODUCTION
```
Comment (MarieEtienne) [inline OCR_pollen.qmd:19]: please use markdown formatting using # for section title and  ## for subsection and so on 
this is not a script file with comments.

**Pull request merged**